lib/wick-test-for-argument-count

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Test to ensure there are a sufficient number of non-empty arguments.
+#
+# $1    - Number of elements that should be in $2.
+# $2-@  - Command line arguments to parse.  Typically this is `"$@"`.
+#
+# Examples
+#
+#   # Ensure the $@ has 4 non-empty arguments.
+#   wickTestForArgumentCount 4 "$@"
+#
+#   # This would fail the test and return 1.
+#   array=(first "" third)
+#   wickTestFormArgumentCount 3 "${array[@]}"
+#
+# Returns zero if arguments are present, one if not.
+wickTestForArgumentCount() {
+    local args count
+
+    wickGetArgument count 0 "$@"
+    shift
+
+    # Ensure count is there and a positive number.
+    if [[ -z "$count" ]] || [[ ! "$count" =~ ^[0-9]+$ ]]; then
```

Comment:
I'd prefer the second bit to be `[[ "$count" -ge 0]]`.  If Bash evaluates it to zero then I'm ok, even if the string is "0xxx".

If you wanted to make sure this is a positive number then use `-gt` instead of `-ge` in my example.  Otherwise you should change the comment above.